tests/functional/lib/helpers.js

```diff
@@ -246,6 +246,19 @@ define([
           .execute(function (verificationLink, windowName) {
             var newWindow = window.open(verificationLink, windowName);
 
+            // Hook up the new window to listen for WebChannel messages.
+            // XXX TODO: this is pretty gross to do universally like this...
+            // XXX TODO: it will go away if we can make the original tab
+            //           reliably be the one to complete the oauth flow.
```

Comment:
We prefer the original sign-up tab to finish the flow, since it's the one holding the encryption material.  It doesn't come into play in this PR, but in a future iteration we want to avoid writing the encryption material to localStorage.  In that case, the verification link would have to re-prompt for your password to complete the flow, while the originating tab could do it automatically.  Hence the (future-looking) preference for completing in the original tab.